### PR TITLE
Add alt-text to all images in Qiskit docs

### LINKED ISSUE #196: Add alt-text to all images in Qiskit docs
Qiskit docs contain 53 images and 0 images are associated with alt text. 

**To-do**: Add alt-text to images in the following files.

```
..  image::
    :alt: alternate text
```

**Aer**

- [x] device_noise_simulation.rst (3)

- [x] simulate_experiments.rst (2)

- [x] simulate_statevectors.rst (2) 

**Qiskit**

- [x] getting-started.rst (6)

- [x] install.rst (2)

- [x] overview.rst (1)

**Ignis**

- [x] overview.rst (1)

**Terra**

- [x] plotting_data_in_qiskit.rst (14)

- [x] quantum_circuits.rst (12)

- [x] visualizing_a_quantum_circuit.rst (10)

### PR DESCRIPTION
**Pull request by derivation**
fixes #196

### CODE REVIEW

File: docs/terra/plotting_data_in_qiskit.rst [hunk 1]
```diff
@@ -265,7 +271,8 @@ statevectors)
 
 
 
-.. image:: ../images/figures/plotting_data_in_qiskit_21_0.png
+.. image:: ../images/figures/plotting_data_in_qiskit_21_0.
```
Comment (lzdanski) [inline docs/terra/plotting_data_in_qiskit.rst]:
```suggestion
.. image:: ../images/figures/plotting_data_in_qiskit_21_0.png
```

Comment (lzdanski) [reply docs/terra/plotting_data_in_qiskit.rst]: Lost the filetype?

**Review by lzdanski — DISMISSED** [review]
SUPER! Thank you so much for doing this! At some point, we might revisit because the alt-text you've composed look like they could be image captions. If you have appropriately descriptive captions, then alt-text isn't required, but when I was going through the docs, we didn't have either. 

For now, this is exactly what was needed. I found one line that might have lost the filetype, but other than that, LGTM.

File: docs/terra/plotting_data_in_qiskit.rst (hunk 1 above)
Comment (derivation) [reply docs/terra/plotting_data_in_qiskit.rst]: good catch!

**Pull request merged**